solutions/adaptive-image/README.md

```diff
@@ -163,8 +163,8 @@ To use adaptive image as an element's background, you can use `is-background-for
 
 ## Development mode (relative image path)
 
-When you developing your application you can use some local development server and relative paths in your project structure for the images. In that case, Uploadcare Proxy service would be disabled for you development environment and you will see original local images in your application until you deploy it:
-
+When you develop your application, you can use some local development servers and relative paths in your project structure for the images. In that case, Uploadcare Proxy service would be disabled for your development environment, and you will see original local images in your application until you deploy it:
```

Comment:
Why "servers"?

Reply:
If "use some local development servers" - you need the plural. If there is only one server, then "some" should be removed.
 
Changed to "use a local development server"